api-description/forms/pk/get.json

```diff
@@ -0,0 +1,60 @@
+{
+    "description": "A detail view for a form",
+    "url": "http://localhost:8000/forms/1",
+    "OK": {
+        "status": 200,
+        "response_data": {
+            "name": "How have you been using the site?",
+            "url": "http://localhost:8000/forms/1",
+            "fieldsets": [
+                {
+                    "name": "Free text field",
```

Comment:
We need either `help_text` or `description` for each `fieldset`. It would help the user to display generic information about the group of fields if necessary.

see 	https://incuna-pm.atlassian.net/wiki/display/OP/Questionnaires

> 003	Each fieldset will optionally include a multiple line free text description.